### PR TITLE
Update SignalR sample

### PR DESCRIPTION
**Pull request by Y-Sindo**
### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

**Comment by Y-Sindo** [comment]
Hi @kshyju, I have added some SignalR samples to demonstrate the usage of new POJO classes added in #774, also to fix the [doc](https://github.com/MicrosoftDocs/azure-docs-pr/pull/202744). Could you please take a look at it?

File: samples/Extensions/local.settings.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
     "AzureWebJobsStorage": "",
     "CosmosConnection": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
+    "SignalRConnection": "",
```
Comment (kshyju) [inline samples/Extensions/local.settings.json:7]: Did you mean to have empty string value here?

Comment (Y-Sindo) [reply samples/Extensions/local.settings.json:7]: No, a valid connection string is required to run the sample. I think it's like the `AzureWebJobsStorage` which also leaves an empty string for user to fill.

**Comment by Y-Sindo** [comment]
Hi @kshyju , the check-enforcer seems to be stuck. Do you know how to fix it or find the right person?

**Pull request merged**